packages/mobile/test/values.ts

```diff
@@ -56,7 +56,7 @@ export const mockE164NumberHashWithPepper =
   '0xf6429456331dedf8bd32b5e3a578e5bc589a28d012724dcd3e0a4b1be67bb454'
 
 export const mockE164Number2 = '+12095559790'
-export const mockDisplayNumber2 = '(209) 555-9790'
+export const mockDisplayNumber2 = '+1 209-555-9790'
```

Comment:
Why is this change needed?

Reply:
The existing mock was not used anywhere and when displaying it showed up like this. I just thought it was an oversight, is this not how it's supposed to be formatted? Argentinian numbers have always showed up following this pattern, but maybe american ones are different, although I didn't change anything related to the number formatting.

Reply:
Thanks all good then.